extract/raw_array_cohort_extract.wdl

```diff
@@ -169,10 +171,10 @@ task ExtractTask {
                 -O "~{output_file}" \
                 ~{probe_info_clause} \
                 --project-id "~{project_id}" \
-                --cohort-sample-table "~{fq_cohort_mapping_table}" \
+                --sample-info-table "~{fq_cohort_mapping_table}" \
```

Comment:
did you rebase off of my changes? i think i changed the name to cohort-sample-table. unless it got changed back

Reply:
rebased